compiler/optimizer/LocalAnalysis.cpp

```diff
@@ -714,7 +714,12 @@ bool TR_LocalAnalysisInfo::isCallLike(TR::Node *node) {
 
    if (node->getOpCode().hasSymbolReference())
       {
-      if (node->getSymbolReference()->getSymbol()->isVolatile() || node->getSymbolReference()->getSymbol()->isMethodMetaData())
+      if (node->getSymbolReference()->getSymbol()->isVolatile() || 
+          (node->getSymbolReference()->getSymbol()->isMethodMetaData()
+#if J9_PROJECT_SPECIFIC
```

Comment:
Thanks Julian. 

I wonder if MethodMetaDataSymbol is the right place to add the routine instead of SymbolReference simply because the query is somewhat specific to meta data fields (i.e. which ones are immutable meta data fields). SymbolReference is a generic wrapper for different Symbol kinds and I am not sure if that is the right place to add a query that is quite specific to one type of Symbol.